src/components/sectionHeader.module.scss

```diff
@@ -0,0 +1,12 @@
+$gray: #e7e7e7;
+
+h3 {
```

Comment:
It seems like this actually applies to all h3 on any page that uses this component? Maybe I should just create a class for this instead

Reply:
It might be better to see if you can't find a way to define the SCSS classes globally so we don't have to redefine them in each file we use; I tried briefly but didn't find anything that worked, but maybe you might have some more luck.